docs/examples/sglang/README.md

```diff
@@ -0,0 +1,100 @@
+# Deploy Distributed Inference Service with SGLang and LWS on GPUs
+
+In this example, we demonstrate how to deploy a distributed inference service using LeaderWorkerSet (LWS) with [SGLang](https://docs.sglang.ai/) on GPU clusters.
+SGLang provides native support for distributed tensor-parallel inference and serving, enabling efficient deployment of large language models (LLMs) such as DeepSeek-R1-671B and Meta-Llama-3.1-405B across multiple nodes. This example uses the [meta-llama/Meta-Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct) model to demonstrate multi-node serving capabilities. For implementation details on distributed execution, see the SGLang docs [Run Multi-Node Inference](https://docs.sglang.ai/references/multi_node.html).
+
+## Install LeaderWorkerSet
+
+Follow the step-by-step guide on how to install LWS. [View installation guide](https://github.com/kubernetes-sigs/lws/blob/main/docs/setup/install.md)
+
+## Deploy LeaderWorkerSet of SGLang
+We use LeaderWorkerSet to deploy 2 SGLang model replicas, and each SGLang replica has 2 pods, and 1 GPUs per pod. It using multiple nodes using tensor parallelism, so the tensor parallelism size is 2.
+The leader pod runs the HTTP server.
+Replace the `HUGGING_FACE_HUB_TOKEN` in `lws.yaml` with your own Hugging Face token. Then, use `lws.yaml` to deploy it.
+```shell
+kubectl apply -f lws.yaml
+```
+
+Verify the status of the SGLang pods
+```shell
+kubectl get pods
+```
+
+Should get an output similar to this
+```shell
+NAME         READY   STATUS    RESTARTS   AGE
+sglang-0     0/1     Running   0          3m55s
```

Comment:
How can we make sure they're running in different nodes, can we output with `kubectl get pods -o wide`?

Reply:
Thanks @kerthcet 

It has been changed :-)